src/components/HTMLElement.js

```diff
@@ -1,4 +1,18 @@
 // Define class here
+class HTMLElement {
+  constructor(tag, content) {
+    this.tag = tag;
+    this.content = content;
+  }
+
+  render() {
+    return '<${this.tag}>${this.content}</${this.tag}>';
```

Comment:
Remember when using this syntax you need to begin and end with ` not a regular apostrophe '